{"uid":"68654dc27f3edf9c","name":"test_exactly_one_primary_reported_by_juju","fullName":"tests.integration.test_charm#test_exactly_one_primary_reported_by_juju","historyId":"e9bc3b3a94e9336e3692785612fb667f","time":{"start":1728788628430,"stop":1728788628430,"duration":0},"description":"Tests that there is exactly one replica set primary unit reported by juju.","descriptionHtml":"<p>Tests that there is exactly one replica set primary unit reported by juju.</p>\n","status":"skipped","statusMessage":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_setup\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 840, in pytest_runtest_setup\n    yield from self._runtest_for(item, \"setup\")\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 875, in pytest_runtest_setup\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_setup\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 160, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 514, in setup\n    col.setup()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1630, in setup\n    self._request._fillfixtures()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 697, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 532, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 617, in _get_active_fixturedef\n    fixturedef.execute(request=subrequest)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1091, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 234, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 206, in xfail\n    raise XFailed(reason)\n_pytest.outcomes.XFailed: aborted","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1728788008226,"stop":1728788008227,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1728788008227,"stop":1728788008391,"duration":164},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1728788008227,"stop":1728788008227,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1728788628430,"stop":1728788628430,"duration":0},"status":"failed","statusMessage":"_pytest.outcomes.XFailed: aborted\n","statusTrace":"  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 234, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 206, in xfail\n    raise XFailed(reason)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"testStage":{"description":"Tests that there is exactly one replica set primary unit reported by juju.","status":"skipped","statusMessage":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_setup\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 840, in pytest_runtest_setup\n    yield from self._runtest_for(item, \"setup\")\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 875, in pytest_runtest_setup\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_setup\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 160, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 514, in setup\n    col.setup()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1630, in setup\n    self._request._fillfixtures()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 697, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 532, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 617, in _get_active_fixturedef\n    fixturedef.execute(request=subrequest)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1091, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 234, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 206, in xfail\n    raise XFailed(reason)\n_pytest.outcomes.XFailed: aborted","steps":[],"attachments":[{"uid":"ed84f03fac362bec","name":"log","source":"ed84f03fac362bec.txt","type":"text/plain","size":2587}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"ops_test::finalizer","time":{"start":1728788628431,"stop":1728788628788,"duration":357},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1728788628789,"stop":1728788628789,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1728788628789,"stop":1728788628789,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1728788628789,"stop":1728788628789,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"group(1)"},{"name":"tag","value":"runner(['self-hosted', 'linux', 'X64', 'jammy', 'large'])"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_charm"},{"name":"host","value":"github-runner"},{"name":"thread","value":"17559-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_charm"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"d0525e98f1493b8","status":"unknown","time":{}}],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":6,"passed":68,"unknown":2,"total":76},"items":[{"uid":"acafdd10110fb633","reportUrl":"../2347//#testresult/acafdd10110fb633","status":"passed","time":{"start":1728702354576,"stop":1728702422368,"duration":67792}},{"uid":"87caf1e0c73d4e16","reportUrl":"../2346//#testresult/87caf1e0c73d4e16","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1728615326347,"stop":1728615326347,"duration":0}},{"uid":"5231bd5f43a30c21","reportUrl":"../2342//#testresult/5231bd5f43a30c21","status":"passed","time":{"start":1728529820498,"stop":1728529885009,"duration":64511}},{"uid":"f64633f522cc3f3e","reportUrl":"../2340//#testresult/f64633f522cc3f3e","status":"passed","time":{"start":1728442680418,"stop":1728442749981,"duration":69563}},{"uid":"cb4d7e7fbd50ff5b","reportUrl":"../2338//#testresult/cb4d7e7fbd50ff5b","status":"passed","time":{"start":1728355161836,"stop":1728355229621,"duration":67785}},{"uid":"870a8eb214bb718f","reportUrl":"../2335//#testresult/870a8eb214bb718f","status":"passed","time":{"start":1728269441237,"stop":1728269507108,"duration":65871}},{"uid":"9e60eb7066bb81ac","reportUrl":"../2334//#testresult/9e60eb7066bb81ac","status":"passed","time":{"start":1728182604803,"stop":1728182672673,"duration":67870}},{"uid":"452b9ad973c193c6","reportUrl":"../2333//#testresult/452b9ad973c193c6","status":"passed","time":{"start":1728097256035,"stop":1728097320533,"duration":64498}},{"uid":"cf89cc62c95ef6bd","reportUrl":"../2330//#testresult/cf89cc62c95ef6bd","status":"passed","time":{"start":1728010017776,"stop":1728010082156,"duration":64380}},{"uid":"b5f062c9c87f06f7","reportUrl":"../2320//#testresult/b5f062c9c87f06f7","status":"passed","time":{"start":1727922523435,"stop":1727922587921,"duration":64486}},{"uid":"6f4cb41592592853","reportUrl":"../2310//#testresult/6f4cb41592592853","status":"passed","time":{"start":1727838197148,"stop":1727838265012,"duration":67864}},{"uid":"d22b42d0ab887fe9","reportUrl":"../2307//#testresult/d22b42d0ab887fe9","status":"passed","time":{"start":1727750800754,"stop":1727750869110,"duration":68356}},{"uid":"992bbfbe43e9e152","reportUrl":"../2304//#testresult/992bbfbe43e9e152","status":"passed","time":{"start":1727665752295,"stop":1727665818614,"duration":66319}},{"uid":"13b73e759389d787","reportUrl":"../2303//#testresult/13b73e759389d787","status":"passed","time":{"start":1727577077985,"stop":1727577144397,"duration":66412}},{"uid":"99289d6c0d7dab0","reportUrl":"../2302//#testresult/99289d6c0d7dab0","status":"passed","time":{"start":1727491243572,"stop":1727491309982,"duration":66410}},{"uid":"3bbcf36a4683d825","reportUrl":"../2300//#testresult/3bbcf36a4683d825","status":"passed","time":{"start":1727406455048,"stop":1727406521471,"duration":66423}},{"uid":"8471c5cc6a8cb2a7","reportUrl":"../2299//#testresult/8471c5cc6a8cb2a7","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1727320015416,"stop":1727320015416,"duration":0}},{"uid":"aa5397a6a24b1996","reportUrl":"../2298//#testresult/aa5397a6a24b1996","status":"passed","time":{"start":1727232568175,"stop":1727232634568,"duration":66393}},{"uid":"a9a8520b16910056","reportUrl":"../2297//#testresult/a9a8520b16910056","status":"passed","time":{"start":1727145748493,"stop":1727145816966,"duration":68473}},{"uid":"f037c5ab778cddaf","reportUrl":"../2293//#testresult/f037c5ab778cddaf","status":"passed","time":{"start":1727059069453,"stop":1727059135869,"duration":66416}}]},"tags":["runner(['self-hosted', 'linux', 'X64', 'jammy', 'large'])","asyncio","group(1)"]},"source":"68654dc27f3edf9c.json","parameterValues":[]}